exporter/sentryexporter/sentry_exporter.go

```diff
@@ -329,9 +331,16 @@ func transactionFromSpan(span *sentry.Span) *sentry.Event {
 // CreateSentryExporter returns a new Sentry Exporter.
 func CreateSentryExporter(config *Config, params component.ExporterCreateParams) (component.TracesExporter, error) {
 	transport := newSentryTransport()
-	transport.Configure(sentry.ClientOptions{
+
+	clientOptions := sentry.ClientOptions{
 		Dsn: config.DSN,
-	})
+	}
+
+	if config.Insecure {
+		clientOptions.HTTPTransport = &http.Transport{TLSClientConfig: &tls.Config{InsecureSkipVerify: true}}
```

Comment:
@bogdandrutu This is a slight aside from the PR, but what do you think about a common `HTTPTransport` that the collector passes to each exporter? I'm sure each exporter is leveraging `net/http` internally, so maybe it makes sense for this to be collector wide config, and then the exporters just inherit whatever `HTTPTransport` that the collector sets.

This also means we don't have to expose the same transport options (for certificates, ssl etc.) over and over again for each exporter the user uses.

Reply:
I think this is a bad idea. 
For example, for some exporter, ssl validation should be ignored and Sentry is used as another exporter. I mean sentry.io which has a completely valid certificate and it would be stupid not to check it. This is just an example of a case where different settings may be required.